assets/javascripts/main.js

```diff
@@ -1,17 +1,28 @@
 require([
+    'utils/ajax',
     'modules/checkout/checkout',
     'modules/digitalpack',
-    'utils/ajax'
+    // Add new dependencies ABOVE this
+    'raven'
 ], function(
+    ajax,
     checkout,
-    digitalpack,
-    ajax
+    digitalpack
 ) {
     'use strict';
 
-    // Global
+    /**
+     * Set up Raven, which speaks to Sentry to track errors
+     */
+    /*global Raven */
+    Raven.config('https://6dd79da86ec54339b403277d8baac7c8@app.getsentry.com/47380', {
```

Comment:
With the current setup, do we capture errors also in our development environments?